internal/clients/wit.go

```diff
@@ -80,7 +84,7 @@ func (wi *WITInfo) UnmarshalJSON(b []byte) (err error) {
 }
 
 // SearchCodebase finds and returns owner of a given repository based on URL.
-func (w WIT) SearchCodebase(repo string) (*WITInfo, error) {
+func (w *wit) SearchCodebase(repo string) (*WITInfo, error) {
```

Comment:
Does SearchCodebase mutate? if not can we keep the receiver a value i.e  `w wit` ?

Reply:
Apparently not, but maybe `w.client.Do` modifies something internally but @chmouel might have the rightanswer

Reply:
we need it for the test and the increment counter,